StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift

```diff
@@ -216,7 +224,9 @@ extension NativeFlowController {
                     switch result {
                     case .success(let session):
                         let eventType = "object"
-                        if session.accounts.data.count > 0 || session.paymentAccount != nil || session.bankAccountToken != nil {
+                        if !session.accounts.data.isEmpty || session.paymentAccount != nil
```

Comment:
this is important to double check

Reply:
Looks good, the only change is .count > 0 vs !isEmpty